src/_utils/item/index.tsx

```diff
@@ -65,7 +74,7 @@ const Item = ({
       tagProps = { ...tagProps, href }
     }
   }
-  const isClickable = !!href || !!onClick
+  const isClickable = !!href || !!onClick || !!onMouseDown || tag.type === 'label'
```

Comment:
not sure to understand why adding `onMouseDown` and `tag.type === label`. It feels hacky, and easy to forget a use case.

Reply:
Yeah, I was hesitating to create a new prop for handling the hover effect (instead of using other props).

`onMouseDown` is for the autocomplete items (that do not use `href` or `onClick`) and `label` is for checkboxes and radios.

Reply:
I switched to the new isClickable prop. More stable than relying on something else.